exercises/concept/lucas-numbers/.meta/exemplar.rb

```diff
@@ -0,0 +1,26 @@
+class LucasNumbers
+  attr_reader :sequence
+
+  def initialize
+    @sequence = create_sequence
+  end
+
+  def to_a(count)
+    raise ArgumentError, "count must be a number greater than or equal to 0" if !count.is_a?(Integer) || count < 1
```

Comment:
```suggestion
    raise ArgumentError, "count must be a number greater than or equal to 0" unless count.is_a?(Integer) && count >= 0
```
The words say that it "must be a number greather than or equal to 0 or equal to 0"  This guard almost identically says the same thing.  But, "a number" is not the same as Integer, so if this must be an integer and not a rational or float, then we could state as much.

Reply:
```suggestion
    raise ArgumentError, "count must be an integer greater than or equal to 0" unless count.is_a?(Integer) && count >= 0
```